src/stores/DocumentsStore.ts

```diff
@@ -0,0 +1,146 @@
+import { Action, createHook, createStore } from "react-sweet-state";
+import documentsApi, { TypeGetParser, TypeKeyValueParser } from "../api/documentsApi";
+import openNotificationWithIcon from "../components/Notifications/Notification";
+import DocumentsTableInfo from "../models/Documents/DocumentsTableInfo";
+
+type State = {
```

Comment:
Is there anu model to use? Instead of writing down plane object multiple times.

If there's no model (class / type / interface), create it.

Maybe it is `CityDocuments`, but I'm not sure.